compose/install/index.md

```diff
@@ -2,31 +2,45 @@
 description: How to install Docker Compose
 keywords: compose, orchestration, install, installation, docker, documentation
 title: Install Docker Compose
-toc_max: 2
+toc_max: 3
+redirect_from:
+- /compose/compose-desktop/
 ---
 
-On this page you can find a summary of the available options for installing Docker Compose. 
+On this page you can find information on how to get and install Compose.
 
-## Compose prerequisites
+## Install Compose
 
-* Docker Compose requires Docker Engine.
-* Docker Compose plugin requires Docker CLI.
+If you have Docker Desktop, you've got a full Docker installation, including Compose.
 
-## Compose installation scenarios
-You can run Compose on macOS, Windows, and 64-bit Linux. Check what installation scenario fits your needs. 
+You can check this by clicking on **About Docker Desktop** from the Docker menu ![whale menu](../../desktop/images/whale-x.png){: .inline}.
 
-Are you looking to:
+![about-desktop](../images/about_desktop.png){:width="750px"}
```

Comment:
Have you checked if we already have an "about desktop" screenshot somewhere? I'm wondering if there's already one somewhere that we can reuse; perhaps it would be good to have this screenshot within the `/desktop/images/` directory to encourage it to be re-used.

My concern with this particular screenshot is that (because it shows "versions"), we may get tempted to updated it with each release (which we should avoid if possible); perhaps we should consider blurring the versions to make it more stand out as "illustrative", and not "here's the version you should see".

Reply:
1) I searched in the most obvi places desktop pages, and sources: desktop, get-started and root folder and didn't find any similar image.
2) I think we don't necessarily need to update it every release, depends on which solution we find acceptable. Maybe we could add a note/sentence saying this is an example for illustration purposes only taken at version x and is not intended to reflect the current state of affairs? WDYT? 
To blur it I need to add this to the backlog of a posterior round of edits.